pkgs/by-name/cs/csfml/package.nix

```
@@ -1,28 +1,37 @@
-{ lib, stdenv, fetchFromGitHub, cmake, sfml }:
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  sfml,
+}:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "csfml";
-  version = "2.5.2";
+  version = "2.6.1";
+
   src = fetchFromGitHub {
     owner = "SFML";
-    repo  = "CSFML";
-    rev   = version;
-    sha256 = "sha256-A5C/4SnxUX7mW1wkPWJWX3dwMhrJ79DkBuZ7UYzTOqE=";
+    repo = "CSFML";
+    rev = finalAttrs.version;
+    hash = "sha256-ECt0ySDpYWF0zuDBSnQzDwUm4Xj4z1+XSC55D6yivac=";
   };
+
   nativeBuildInputs = [ cmake ];
   buildInputs = [ sfml ];
   cmakeFlags = [ "-DCMAKE_MODULE_PATH=${sfml}/share/SFML/cmake/Modules/" ];
+  patches = [ ./pkgconfig.patch ];
 
```

Lets split the package between `out` & `dev`
```suggestion

  outputs = [ "out" "dev" ];
```